accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,102 @@
+NOTE: `skuba-update` is **NOT** invoked by Uyuni/SUSE Manager and it is running autonomously on the nodes of the CaaS Platform cluster.
+
+For example, an organization could have a "staging" cluster with a set of staging channels. The operator would cherry-pick Kubernetes updates from the SCC channels into the staging ones and then wait for skuba-update to roll them out into the staging cluster. At a later time, once testing is green inside of the staging cluster, the packages can be promoted to the "production channels" and be rolled out to the production cluster.
+
+It is paramount to state in the user interface and the documentation that the information presented for the minion is in read-only mode.
+If a user reboots the node, installs a patch or installs a package related to Kubernetes (`kubernetes-kubeadm kubernetes-kubelet kubernetes-client cri-o cni-plugins`) the Kubernetes may break.
+
+NOTE: both of the above actions will not cluster-coordinated, so it is entirely possible that a user assigns different channels, install packages or reboots nodes in a subset of nodes belonging to the same cluster.
+
+## Update the current Kubernetes integration
+
+Under Virtual Host Manager, there is an already Kubernetes integration that supports CaaS Platform version 3.
+The current implementation breaks with CaaS Platform version 4: the Salt runner needs to take into account that CaaS Platform version 4 switched from Docker to CRI-O.
+Uyuni/SUSE Manager must check if the underlying container engine is running Docker (and use the existing Kubernetes integration) or CRI-O (and adapt the current implementation to support CRI-O).
+The Kubernetes implementation that this RFC is targeting is the one in CaaS Platform version 4: 1.15.2. Other Kubernetes distributions will not be targeted, at least initially.
+
+Into the Virtual Host Manager, when a Kubernetes cluster is selected, the properties and the names of the nodes are shown, together with CPU Arch, CPU Sockets and RAM of the node.
+
+After the user has imported the `kubeconfig`, obtained from CaaS Platform, the process would be:
+
+- When a user registers a new cluster, we introduce a new workflow to bootstrap (using Systems > Bootstrapping) each node of the cluster
+- When every cluster node is registered, a system group with the same name of the cluster is created and all registered systems are moved into this system group
+- Each node presented the Nodes information is a link to the system overview page of that node
+
+## Documentation
+
+In addition to all the above, we are enriching our documentation on the CaaS Platform deployment. In the following sections, we are going to describe what we should introduce in the documentation.
+
+### AutoYaST Deployment documentation
+
+CaaS Platform provides AutoYaST templates for deploying a standard cluster (in the `SUSE-CaaSP-Management` package) comprising of:
+
+- 2 Load Balancers
+- 3 Masters
+- 3 Workers
```

Comment:
The [minimum supportable cluster](https://github.com/SUSE/doc-caasp/blob/master/adoc/deployment-sysreqs.adoc) is 3 masters and 2 workers.

Reply:
Fixed (this might need to be corrected on the CaaSP side https://documentation.suse.com/suse-caasp/4/single-html/caasp-deployment/#_provisioning_the_cluster_nodes)